nautobot/ipam/api/serializers.py

```diff
@@ -165,6 +166,32 @@ class Meta:
             "prefix_length": {"read_only": True},
         }
 
+        detail_view_config = {
+            "layout": [
+                {
+                    "Prefix": {
+                        "fields": [
+                            "prefix",
+                            "parent",
+                            "namespace",
+                            "type",
+                            "network",
+                            "broadcast",
+                            "prefix_length",
+                            "ip_version",
+                            "description",
+                            "role",
+                            "vlan",
+                            "location",
+                            "tenant",
+                            "rir",
+                            "date_allocated",
+                        ]
+                    },
+                },
```

Comment:
Do we need to add this to get tags to appear for now?

```
              {"Tags": {"fields": ["tags"]}}
```

Reply:
I tweaked `validated_view_config` method to add `Tags` field automatically if it is not added already.

Reply:
I'm doing something very similar in #4532 as part of a broader refactor, let's stay in sync here.